pkg/agent/util/net.go

```diff
@@ -415,3 +415,17 @@ func GenerateRandomMAC() net.HardwareAddr {
 	buf[0] |= 2
 	return buf
 }
+
+func GetGlobalIPNetsByName(link *net.Interface) ([]*net.IPNet, error) {
```

Comment:
It feels like the name of this function is not accurate since there is no `name` parameter...